mysql-test/suite/binlog_encryption/t/encrypted_master_switch_to_unencrypted.cnf

```diff
@@ -0,0 +1,5 @@
+!include ../my.cnf
```

Comment:
Orphaned file?

Reply:
I have restored this test file. Seems I had a problem with this test because I had not crc check implemented yet. Now it works.